RedHatInsights/rhsm-subscriptions
# SWATCH-2170: Display the host name on the health endpoints for Quarkus

Jira issue: [SWATCH-2170](https://issues.redhat.com/browse/SWATCH-2170)

## Description
- Rename SPLUNKMETA_host to HOST_NAME
- Create common health module

## Testing
- podman-compose up
- start any of the Quarkus services, for example: `./gradlew :swatch-metrics:quarkusDev`
- see the health check endpoint: `curl localhost:9000/health`

You should see the hostname as:

```json
{
    "status": "UP",
    "checks": [
        {
            "name": "info",
            "status": "UP",
            "data": {
                "host": "jcarvaja@fedora"
            }
        },
...
```
